mod/wikirate_source/spec/set/type/source/source_spec.rb

```diff
@@ -109,10 +109,10 @@
         end
       end
     end
-    describe "while creating a source with a wikirate link" do 
+    describe "while creating a source with a wikirate link" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.